app/Casts/TagArrayCast.php

```
@@ -8,48 +8,53 @@
-	 * @param Model                $model      the associated model class
-	 * @param string               $key        the name of the SQL column holding the stringified array
-	 * @param (string|null)[]|null $value      the array
-	 * @param array<string,mixed>  $attributes
+	 * @param Model               $model      the associated model class
+	 * @param string              $key        the name of the SQL column holding the stringified array
+	 * @param (Tag|null)[]|null   $value      the array
+	 * @param array<string,mixed> $attributes
 	 *
 	 * @return array<string,mixed> An associative map of SQL columns and their values
 	 */
 	public function set(Model $model, string $key, mixed $value, array $attributes): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (Unused Code Rules)

(UnusedFormalParameter)